docker-compose.yml

```diff
@@ -32,6 +32,9 @@ services:
       - ./edx-platform/config/docker_run_lms.py:/config/docker_run_lms.py
       - ./edx-platform/lms/urls.py:/app/edx-platform/lms/urls.py
       - .:/app/fonzie
+      - ./data/static/lms:/data/static/lms
+      - ./data/media:/data/media
+      - ./data/logs:/data/logs
```

Comment:
You ca use a tmpfs
https://docs.docker.com/compose/compose-file/#tmpfs

Reply:
Done.